spk/cloudflared/Makefile

```diff
@@ -0,0 +1,23 @@
+SPK_NAME = cloudflared
+SPK_VERS = 2022.6.3
+SPK_REV = 1
+SPK_ICON = src/cf-logo-v-rgb.png
```

Comment:
I recommend to change the icon file name to match the package name.

Reply:
Done